disnake/ui/action_row.py

```diff
@@ -329,7 +330,8 @@ def add_button(
 
             .. versionadded:: 2.11
         id: :class:`int`
-            The numeric identifier for the component.
+            The numeric identifier for the component. Must be unique within a message.
+            This is always present in components received from the API.
```

Comment:
A little pedantic, but the `This is always present in components received from the API` part is perhaps not as useful when talking about parameters of these `add_*` methods.